docs/pages/contributing.md

```diff
@@ -176,3 +176,7 @@ available at [http://contributor-covenant.org/version/1/4][version]
 [gatsby]: https://www.gatsbyjs.org/
 [repo]: https://github.com/ifiokjr/remirror
 [husky]: https://github.com/typicode/husky
+
+### Troubleshooting
+
+If you're getting errors like `ReferenceError: CodeBlockExtension is not defined` but you know you've imported it, it might be because you've not added it as a dependency to the relevant `package.json`. Rather than throwing an error in this case, rollup (?) seems to just drop the import statement but still persist the lines where the import is used.
```

Comment:
This had me scratching my head for a good while! Is it rollup doing this, or...?

Reply:
I ran into the problem a while back. I think I fixed it changing `export { Value } from './file` to `export * from './file'`. I don't think the babel build can currently determine which items to export from a typescript file since it's not sure which is a type and which is an actual value (when directly exporting).

I could be completely wrong though.

Reply:
Interesting. For me, adding the dependencies to package.json solved it.

Reply:
Brilliant! 😄 

I might be conflating two unrelated issues to be honest.